reservations/migrations/0002_auto_20201118_0912.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 3.0.10 on 2020-11-18 09:12
```

Comment:
I suggest we use same system as we used with Respa with migration naming.

Leave the number at start as is and after that give it a describing name for the migrations for example:

0002_add_fields_to_recurring_reservations.py